### PR TITLE
[DO NOT MERGE] Add Do you have friends or family to talk to question

### PR DESCRIPTION
**Pull request by huwd**
As of 13:55 this question has been crossed through...

# Screenshots
image

image

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -43,23 +43,25 @@
...
 
   # (v4[sunday]) Question 13: Do you have any special dietary requirements?
   get "/coronavirus-form/dietary-requirements", to: "coronavirus_form/dietary_requirements#show"
-  post "coronavirus-form/dietary-requirements", to: "coronavirus_form/dietary_requirements#submit"
+  post "/coronavirus-form/dietary-requirements", to: "coronavirus_form/dietary_requirements#submit"
 
   # (v4[sunday]) Question 14: Is there someone in the house who's able to carry a delivery of supplies inside?
   get "/coronavirus-form/carry-supplies", to: "coronavirus_form/carry_supplies#show"
-  post "coronavirus-form/carry-supplies", to: "coronavirus_form/carry_supplies#submit"
+  post "/coronavirus-form/carry-supplies", to: "coronavirus_form/carry_supplies#submit"
```
Comment (huwd) [inline config/routes.rb:64]: Sorry, also snuck these in here - i think they all need to begin with "/"

**Comment by leenagupte** [comment]
@huwd I think this question is being merged with the basic care question, so we should hold off on merging.

**Comment by issyl0** [comment]
This question doesn't appear in the v5 protype.

**Comment by huwd** [comment]
:sob:

**Pull request closed without merging**